CMakeLists.txt

```diff
@@ -270,7 +275,7 @@ endif()
 add_subdirectory(idd)
 
 execute_process(
-  COMMAND ${PYTHON_EXECUTABLE} "${PROJECT_SOURCE_DIR}/scripts/dev/generate_epJSON_schema/generate_epJSON_schema.py" "${PROJECT_SOURCE_DIR}"
+  COMMAND ${Python_EXECUTABLE} "${PROJECT_SOURCE_DIR}/scripts/dev/generate_epJSON_schema/generate_epJSON_schema.py" "${PROJECT_SOURCE_DIR}"
```

Comment:
`PYTHON_XXXX` variables were retained for backward compatibility, but the official casing is `Programname_XXX` so `Python_XXX` to be consitent with the rest of cmake.

Reply:
OK, makes sense.  I'm pretty sure if anyone ever adds another call from Python they'll end up just copying whatever we have in place anyway.